docs/manifests/hostnetwork/elastic-agent-standalone-daemonset-ksm-0.yaml

```diff
@@ -0,0 +1,841 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: agent-node-datastreams
+  namespace: kube-system
+  labels:
+    k8s-app: elastic-agent
+data:
+  agent.yml: |-
+    id: 439ab9f0-f010-11ed-a741-bb715e0e238e
```

Comment:
Why do you need an id here?
This looks like it is copied from Fleet UI, right ?
(the same question applies to the other configmaps which contain the elastic-agent.yml with a specific id)

Reply:
Yes it is copied from there. Do you think shall I remove them?

Reply:
removed them

Reply:
I was asking because it is weird to me that in a config shared among multiple instances we specify the same id (if it's not there each agent should generate its own and save it along with the agent state)